tests/Neo.Network.RPC.Tests/UT_RpcServer.cs

```diff
@@ -0,0 +1,29 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Neo.Plugins;
+using Neo.SmartContract.Native.Tokens;
+using Neo.VM.Types;
+
+namespace Neo.Network.RPC.Tests
+{
+    [TestClass]
+    public class UT_RpcServer
+    {
+        [TestMethod]
+        public void Test_ConvertIEnumeratorToArray()
+        {
+            StackItem[] stackItems = new StackItem[]
+            {
+                12345,
+                "hello",
+                new InteropInterface(new int[] { 1, 2, 3 }.GetEnumerator()),
+                new InteropInterface(new Nep5AccountState[] {new Nep5AccountState()}.GetEnumerator())
```

Comment:
Could add a `StorageMap` type here, which used in nep11, see https://github.com/neo-project/neo-devpack-dotnet/pull/268/